src/test/java/poll/service/VoteServiceTest.java

```diff
@@ -1,4 +1,78 @@
 package poll.service;
 
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import poll.repository.Answer;
+import poll.repository.Vote;
+import poll.repository.VoteRepository;
+
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+
+
+@ExtendWith(MockitoExtension.class)
 public class VoteServiceTest {
+
+    @Mock
+    VoteRepository voteRepository;
+
+    @InjectMocks
+    VoteService voteService;
+
+
+    @Test
+    public void should_ReturnVote_when_GivenVoteId() {
+        //given
+        Vote expectedVote = new Vote(3l, "Ivan Zaitsev", LocalDateTime.of(2021, 1, 2, 9, 30), Answer.YES);
+        //when
+        when(voteRepository.findById(3l)).thenReturn(Optional.of(expectedVote));
+        //then
+        assertEquals(voteService.getVoteById(3l), expectedVote);
```

Comment:
The parameters are in the wrong order :D 
The assertEquals() method's declaration is: 
`assertEquals​(Object expected, Object actual)`
So 1st parameter should be the expected object - expectedVote

It matters a lot - when the test fails, it gives the output message with expected and actual values